constants_local.xml

```diff
@@ -190,12 +190,18 @@
         <CONNECTION_ALL_TIMEOUT>1</CONNECTION_ALL_TIMEOUT>
         <!-- Timeout in seconds for ONLY connection that reach our callback function, 0 means no timeout-->
         <CONNECTION_CALLBACK_TIMEOUT>0</CONNECTION_CALLBACK_TIMEOUT>
-        <ENABLE_EVM>false</ENABLE_EVM>
+        <ENABLE_EVM>true</ENABLE_EVM>
         <EVM_SERVER_BINARY>/usr/local/bin/evm-ds</EVM_SERVER_BINARY>
         <EVM_SERVER_SOCKET_PATH>/tmp/evm-server.sock</EVM_SERVER_SOCKET_PATH>
         <EVM_LOG_CONFIG>/usr/local/etc/log4rs.yml</EVM_LOG_CONFIG>
         <!-- eth chain id : Test net = 0x814d, main net: 0x8001 -->
         <EVM_ZIL_SCALING_FACTOR>1000000</EVM_ZIL_SCALING_FACTOR>
+        <!-- blocking Call to read socket from evm-ds timeout -->
+        <EVM_RPC_TIMEOUT_SECONDS>59</EVM_RPC_TIMEOUT_SECONDS>
+        <!-- manage the evm-ds instance or not -->
+        <LAUNCH_EVM_DAEMON>true</LAUNCH_EVM_DAEMON>
+        <!-- Sort account store delta before serialization -->
+        <SORT_ACC_STORE_DELTA>false</SORT_ACC_STORE_DELTA>
```

Comment:
I think we removed this in https://github.com/Zilliqa/Zilliqa/pull/3343

Reply:
Good catch. Updated